jina/types/document/__init__.py

```diff
@@ -142,6 +143,8 @@ class Document(ProtoTypeMixin):
 
     """
 
+    ON_GETATTR = ['matches']
```

Comment:
it seems weird that matches is in a list where chunks is not. Why is that?

Reply:
oh forgot to add it back,  fixing it

Reply:
fixed